### ISSUE TITLE
no-unbound-method warns on method existence check

### ISSUE THREAD

**Issue by NN---** [palantir/tslint#3009]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: 2.4.1
- __Running TSLint via__: gulp-tslint

#### TypeScript code being linted

```ts
var a = document.addEventListener ? document.addEventListener  : null;
```

#### Actual behavior
No error

#### Expected behavior
error noUnboundMethod: Avoid referencing unbound methods which may cause unintentional scoping of 'this'.

**Comment by adidahiya** [comment]
can you paste the verbose lint failure? which [line, col] is it failing on? it's more likely on the second instance of `document.addEventListener`, which is potentially a legit error (invoking `a()` is not the same as `document.addEventListener()`).

**Comment by NN---** [comment]
This is the code:

```
if (document.documentElement.doScroll) {
}
```

Definition of doScroll:

```
interface HTMLElement {
	doScroll(component: string): boolean;
}
```

TSLint:

```
[16:40:57] Using gulpfile C:\Projects\Agent-9.2\ChromeExtension\gulpfile.js
[16:40:57] Starting 'tslint'...
test.ts(30,6): error noUnboundMethod: Avoid referencing unbound methods which may cause unintentional scoping of 'this'.

[16:41:00] 'tslint' errored after 3.64 s
[16:41:00] Error in plugin 'gulp-tslint'
Message:
    Failed to lint: test.ts [30, 6]: Avoid referencing unbound methods which may cause unintentional scoping of 'this'..
```

**Comment by ajafff** [comment]
Fixed by #3349
